NewForce-Cohort-4/nutshell-team-odd-eclipse
# Dtb events by month

# Description
Recovered main.css. Repaired modal functionality. Fixed events date issues. Created sub-headings to partition events by month.

## Type of change
Please delete options that are not relevant.
# Testing Instructions
- Copy "events" from API.txt and paste into database.json.
- Check to ensure that the events show up with date strings and that they have subheadings to denote months.
- Check that the modal pops up when the "Create Article" or "+ Event" button is clicked. 
# Checklist:
